### PR TITLE
fix: fix broken refs and change warns to errors

### LINKED ISSUE #980: Broken links in stacked_drg.md to client-data.md
There are 2 broken links in `stacked_drg.md` that point to `client-data.md` it is not clear where they should be linked to.

- Filecoin Client Data Processing - client-data.md#storage-format
- Client Data Processing - client-data.md#storage-format

https://github.com/filecoin-project/specs/blob/f40b9428e9283ac62230dd4342559e6224a623fa/next/content/algorithms/porep/stacked_drg.md#replication

### LINKED ISSUE #1103: fix hugo warnings
after fixing the warning make them error so it fails the CI

### PR DESCRIPTION
**Pull request by hugomrdias**
closes #980
closes #1103

### CODE REVIEW

**Review by olizilla — APPROVED** [review]
👍

File: content/algorithms/porep/stacked_drg.md [hunk 1]
```diff
@@ -234,7 +234,7 @@ TODO: link to filproofs/feistel
 
 > The Replication phase turns an *unsealed sector* into a *sealed sector* by first *generating a key*, then using the key to *encode the orignal data*.
 
-Before running the `Replicate` algorithm, the prover must ensure that the sector is correctly formatted with a valid "Storage Format" described in [Filecoin Client Data Processing](/missing_link) (currently `paddedfr32v1` is the required default).
+Before running the `Replicate` algorithm, the prover must ensure that the sector is correctly formatted with a valid "Storage Format" described in Filecoin Client Data Processing (currently `paddedfr32v1` is the required default).
```
Comment (olizilla) [inline content/algorithms/porep/stacked_drg.md:237]: If we have nothing to link to here, we should re-word it to not imply that there is
```suggestion
Before running the `Replicate` algorithm, the prover must ensure that the sector is correctly formatted with a valid "Storage Format" (currently `paddedfr32v1` is the required default).
```

Comment (hugomrdias) [reply content/algorithms/porep/stacked_drg.md:237]: i will update this !

Comment (hugomrdias) [reply content/algorithms/porep/stacked_drg.md:237]: after `hugo: downloading modules ...` ends 😱

**Pull request merged**